src/dbus_api/macros.rs

```diff
@@ -42,3 +42,9 @@ macro_rules! get_mut_pool {
         }
     };
 }
+
+macro_rules! uuid_to_string {
+    ($uuid:expr) => {
+        format!("{}", $uuid.to_simple_ref())
```

Comment:
Also, I like to use ```x.to_string()``` instead of ```format!("{}", x)``` where the format string is just ```{}```, it seems less roundabout.

Reply:
Resolved via #1609